lib/ufe/UsdAttribute.h

```diff
@@ -43,9 +43,9 @@ class UsdAttribute
 	bool hasValue() const;
 	std::string name() const;
 	std::string documentation() const;
-	std::string string() const;
+	std::string string(const Ufe::SceneItem::Ptr& item) const;
 
-public:
+protected:
```

Comment:
Why not private?

Reply:
These members are needed by the derived classes.

Reply:
I looked at derived classes and now I see why it's done this way. Thx.